content/en/blog/2021/simple-vms-part-2/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Part 2 - Implementing an easier way to add virtual machines to Istio service mesh"
+description: How to statically configure envoy to join a mesh.
+publishdate: 2022-01-17
+attribution: "Nick Nellis (Solo.io)"
+keywords: [virtualmachine,Istio,networking,envoy]
+---
+![Virtual Machine](./istio-setup.png)
+
+Part 1 - [An easier way to add virtual machines to Istio service mesh](/blog/2021/simple-vms/)
+
+A lot of Istio users would like to connect their virtual machines to their service mesh but the current process is very cumbersome and hard to automate. This series explains a simpler way to connect outside applications securely to your applications running in a service mesh. If you would like to know more about why this might work for you, check out part 1 here.
```

Comment:
```suggestion
A lot of Istio users would like to connect their virtual machines to their service mesh but the current process is very cumbersome and hard to automate. This series explains a simpler way to connect outside applications securely to your applications running in a service mesh. If you would like to know more about why this might work for you, check out [part 1 here](/blog/2021/simple-vms/).
```